src/ripple/basics/hardened_hash.h

```diff
@@ -59,12 +59,12 @@ make_seed_pair() noexcept
     {
         std::mutex mutex;
         std::random_device rng;
-        std::mt19937_64 gen {rng()};
+        std::mt19937_64 gen;
         std::uniform_int_distribution <std::uint64_t> dist;
 
         state_t() : gen(rng()) {}
-        // state_t(state_t const&) = delete;
-        // state_t& operator=(state_t const&) = delete;
+        state_t(state_t const&) = delete;
+        state_t& operator=(state_t const&) = delete;
```

Comment:
I have the extremely vague recollection that these were commented out because of Visual Studio 2013 build problems.  @HowardHinnant are you hooked up to AppVeyor and does this pass the AppVeyor build?  If VS can handle this then I think we're golden.